modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/ImportStyleBookDisplayContext.java

```diff
@@ -23,48 +23,46 @@ public ImportStyleBookDisplayContext(RenderRequest renderRequest) {
 		_renderRequest = renderRequest;
 	}
 
-	public List<String> getStyleBookEntryZipProcessorImportResultEntryNames(
-		StyleBookEntryZipProcessorImportResultEntry.Status status) {
+	public List<String> getStyleBookEntryImporterImportResultEntries(
+		StyleBookEntryImporterImportResultEntry.Status status) {
 
-		List<StyleBookEntryZipProcessorImportResultEntry>
-			styleBookEntryZipProcessorImportResultEntries =
-				_getStyleBookEntryZipProcessorImportResultEntryNames();
+		List<StyleBookEntryImporterImportResultEntry>
+			styleBookEntryImporterImportResultEntries =
+				_getStyleBookEntryImporterImportResultEntries();
 
-		if (ListUtil.isEmpty(styleBookEntryZipProcessorImportResultEntries)) {
+		if (ListUtil.isEmpty(styleBookEntryImporterImportResultEntries)) {
 			return null;
 		}
 
 		return TransformUtil.transform(
-			styleBookEntryZipProcessorImportResultEntries,
-			styleBookEntryZipProcessorImportResultEntry -> {
-				if (styleBookEntryZipProcessorImportResultEntry.getStatus() !=
+			styleBookEntryImporterImportResultEntries,
+			styleBookEntryImporterImportResultEntry -> {
+				if (styleBookEntryImporterImportResultEntry.getStatus() !=
 						status) {
 
 					return null;
 				}
 
-				return styleBookEntryZipProcessorImportResultEntry.getName();
+				return styleBookEntryImporterImportResultEntry.getName();
 			});
 	}
 
-	private List<StyleBookEntryZipProcessorImportResultEntry>
-		_getStyleBookEntryZipProcessorImportResultEntryNames() {
+	private List<StyleBookEntryImporterImportResultEntry>
+		_getStyleBookEntryImporterImportResultEntries() {
```

Comment:
@algobob a change like this should always be in it's own commit. We want to draw attention to the fact that this was incorrectly named before and that we are fixing it now. If we don't draw attention to it, then it looks like a mistake while refactoring. In a separate commit, the commit message can explain why the change is being made. For example: "Update function name to reflect the return type: .StyleBookEntryImporterImportResultEntry".

Please update the commits so that this is in it's own commit with an informative message.